web/src/test/auto/dom/web-test-runner.CI.config.mjs

```diff
@@ -9,5 +9,6 @@ export default {
   reporters: [
     teamcityReporter(), /* custom-written, for CI-friendly reports */
     sessionStabilityReporter({ciMode: true})
-  ]
+  ],
+  concurrency: 1
```

Comment:
I just noticed that this setting was missing from the 'dom' subfolder while present in the 'integrated' subfolder.  There's no clear cause behind the difference in state when tracing the origin of the lines via `git blame`.  Concurrency is known to reduce stability a bit, so perhaps dropping it in CI mode may help with #12638.

https://github.com/keymanapp/keyman/blob/bbd1dd82f7d736a9c813e6feaed26dbb4f22e112/web/src/test/auto/integrated/web-test-runner.CI.config.mjs#L9